docs/source/concepts/service.rst

```diff
@@ -197,6 +194,25 @@ status codes.
 
         return result
 
+For gRPC endpoints, the `grpc.aio.ServicerContext <https://grpc.github.io/grpc/python/grpc_asyncio.html#grpc.aio.ServicerContext>`_ can be accessed through the inference context at ``ctx.grpc.context``.
```

Comment:
We should probably have a chat about this; I'm not sure I want to expose either the gRPC or HTTP context / requests directly, can we do this in i.e. our IO descriptors?